benchmarks/src/bin/tpch.rs

```diff
@@ -1022,9 +1024,10 @@ mod tests {
             ctx.register_table(table, Arc::new(provider))?;
         }
 
-        let plans = create_logical_plans(&ctx, n)?;
-        for plan in plans {
-            execute_query(&ctx, &plan, false).await?;
```

Comment:
Why not just pass `sql` into this method instead of `plan`. Inside the method you can add a couple lines to get the plan from the SQL. This will retain the existing debug logging.

```
    let plan = ctx.sql(sql).await?;
    let plan = plan.to_logical_plan()?;
```

Reply:
I didn't realize how easily a DataFrame can provide a LogicalPlan - this looks great to me. Thanks!